modules/extensions/process/publisher/process/apis/delete_Predecessor.jag

```diff
@@ -2,11 +2,15 @@
 importPackage(org.wso2.carbon.pc.core);
 
 var log = new Log();
-log.info("Predecessor delete logic");
+if(log.isDebugEnabled()){
+    log.debug("Predecessor delete logic");
+}
 
```

Comment:
Make log messages more descriptive. Also write them in the form "Creating predecessors of process X".